Update README.md (#6319)
umbraco/UmbracoDocs: 14/umbraco-cms/extending/section-trees/sections/README.md

When creating a new section it's recommended to use a [Entry Point](../../backoffice-setup/extension-registry/entry-point.md)-extension in your [Umbraco Package Manifest](../../package-manifest.md). This is to get better control over all the additional extensions required for the new section.

Create a section by defining a manifest for it:

```typescript
{
 "type": "section",
 "alias": "My.Section",
 "name": "My Section",
 "meta": {
  "label": "My.Section",
  "pathname": "my-section"
 }
}
```

Once registred you will be able to pick this action for your User Groups Permissions. And once that is permitted you can see your section:

<figure><img src="../../../.gitbook/assets/section-empty.png" alt=""><figcaption><p>Section</p></figcaption></figure>

#### **Extend with Sidebar, Dashboards and more**

> Once a section is registered it can be extended upon as any other section.

Reviewer: ```suggestion
Once a section is registered, it can be extended like any other section.
```

Here is a list of appropriate extensions to append to your section:

- [Dashboard](../../tutorials/creating-a-custom-dashboard/)
- [Section Sidebar](./section-sidebar) 
- [Section View](./section-view)

#### **Manifest with empty element**

If you prefer full control over the content of your section you can choose to define an element for the content of your section.

{% hint style="warning" %}

> This is not recommended as it limits the content of your section to just this element. Instead its recommended to use a single Dashboard or Section View.

Reviewer: ```suggestion
This is not recommended as it limits the content of your section to this element. Instead, it is recommended to use a single Dashboard or Section View.
```

}
```

> For this to work the element file, must have a `element` or `default` export. Or define the specific element name in the `elementName` field.

Reviewer: ```suggestion
The element file must have an `element` or `default` export, or specify the element name in the `elementName` field.
```
